lib/node_modules/@stdlib/math/stats/anova1/lib/make_anova.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+// MODULES //
+
+var isNumberArray = require( '@stdlib/assert/is-number-array' ).primitives;
+var isTypedArrayLike = require( '@stdlib/assert/is-typed-array-like' );
+var isArray = require( '@stdlib/assert/is-array' );
+var setReadOnly = require( '@stdlib/utils/define-read-only-property' );
+var cdf = require( '@stdlib/math/base/dists/f/cdf' );
+var copy = require( '@stdlib/utils/copy' );
+var defaults = require( './defaults.json' );
+var validate = require( './validate.js' );
+var uniqueElems = require('./unique_elems.js');
+var meanTable = require('./mean_table.js');
+var prettyPrint = require( './print.js' );
+
+
```

Comment:
Before the main function in a file, we always place a `// MAIN // ` header (except for the case when there is nothing coming before the main function).

Reply:
I placed it in `make_anova.js` below the description and above the `anova1` function.